articles/ros2_threat_model.md

```diff
@@ -2496,8 +2499,29 @@ to True</li>
     </td>
     <td class="warning">Risk is mitigated if DDS-Security is configured
 appropriately.</td>
-    <td> </td>
-  </tr>
+    <td>As of writing, XRCE only has implementations for UDP, TCP, and serial
```

Comment:
As mentioned in the other comment, that's not really actionable as we can't really modify the spec. If you start fighting against the spec, it just may not be the right tool for you.

Reply:
The XRCE spec is setup (and recommends) using secure transport layers, but there doesn't appear to be an implementation for these yet (https://micro-xrce-dds.readthedocs.io/en/latest/client.html). But I agree it's not immediately actionable so I removed this recommendation.

Reply:
Oh I see - thanks for looking into this. May be worth it to add exactly this comment in the doc too. This is super useful info to whoever tries to design a new robot system